cloudferrylib/os/network/neutron.py

```diff
@@ -881,6 +894,14 @@ def upload_networks(self, networks):
             LOG.debug("Trying to create network '%s'", net['name'])
             tenant_id = identity.get_tenant_id_by_name(net['tenant_name'])
 
+            if tenant_id is None:
```

Comment:
Actually I think we should simply skip network if tenant is not available on dest, this is default behavior for neutron module in CF.